prQuotes.json

```diff
@@ -43,14 +43,15 @@
         "It's nice when the only people you can disappoint is yourself",
         "If it's make you happy, then it's good"
```

Comment:
This isn't English

Reply:
I know... but Felipe wrote it so I resisted the urge to change things TOO much.